opentelemetry/proto/common/v1/common.proto

```diff
@@ -15,6 +15,7 @@
 syntax = "proto3";
 
 package opentelemetry.proto.common.v1;
+import "options/parquet_options.proto";
```

Comment:
I think this will affect everyone who is not using the docker container to build the protos (using something like a build tool specific for the language, e.g. gradle). Can we ask the maintainers if this will cause them troubles?

Reply:
True, how do we ask maintainers? Is there an alternative we could explore here?

Reply:
Yeah this will break the Java build, and I guess it should be a goal for this repo to at least be able to build with common proto tools, not requiring build-tools just for a simple build of the OTLP protocol.

Since the parquet protoc only implements this functionality in a personal fork, does it make sense to fork it into opentelemetry itself, and let it recognize a `.proto` file within this repo?

Alternatively, `sed` on the outputing schema files to add `(JSON)` to these two lines may be the most practical if avoiding such a fork, hacky but would work I guess

Reply:
@anuraaga happy to push this generator to opentelemetry. It's not a personal fork anymore, the original maintainer stopped helping and I took it on and made releases. It could use (more) tests.

No, you can't just add JSON to the schema lines, this is also stopping the generator from going into a recursive infinite loop.

But we could add the annotations to the proto files via sed if you like, that's a fine way to go around it.

Could we inline the options package into opentelemetry-proto otherwise? No harm if we have that checked in?

Reply:
@atoulme Yeah if the referenced proto file is in this repo then there shouldn't be any issues.

Reply:
It might look kludgey but if you’re ok with it that should be a quick fix.

Reply:
done!